### PR TITLE
Update Move-csuser.cmd

### PR DESCRIPTION
**Pull request by ruchiagrawal9221**
Fixing the purpose and status of "ProxyPool" parameter for the Move-csuser powershell cmdlet.

### CODE REVIEW

**Comment by ruchiagrawal9221** [comment]
#sign-off

File: skype/skype-ps/skype/Move-CsUser.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ Accept wildcard characters: False
 
 ### -ProxyPool
 
-This parameter has been deprecated and should not be used.
+This is an optional parameter can be used to specify the front-end pool for user migration.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Move-CsUser.md:268]:
```suggestion
This is an optional parameter that can be used to specify the front-end pool for user migration.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review - @ruchiagrawal9221, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ruchiagrawal9221, kindly commit the suggested changes. Many thanks.

**Comment by kamfooaw** [comment]
@dariomws and @get-itips, @ruchiagrawal9221 and I are from the same team in Microsoft whuch supports user migration.  She has not signed up yet as Microsoft Doc team.  I made the same change in my PR: #10636.  This PR is a duplicate. Sorry for the confusion.

**Pull request closed without merging**